Pledge-Based Saturation Limit Under a Closed System/README.md

```diff
@@ -0,0 +1,252 @@
+  
+![logic2](https://drive.google.com/uc?export=view&id=1j45ZEEhqwDDJoXcVxs4p1NqixegXGbmM)  
+  
+## **_Sybil Attack_**
+
+We can calculate the total pledge to conduct a Sybil attack using the following equation:
+
+![cost](https://drive.google.com/uc?export=view&id=1cqZJ7y9oBi1bA6rOYWzlseUAIyOH3avg)
+
+We consider a scenario where some pools turn adversarial (**_k<sub>adversarial</sub>_**), and we can give the best scenario for the attackers by assuming that their pools are fully saturated. We can then plot the total pledge to conduct a Sybil attack as well as **_k<sub>adversarial</sub>_** vs the number of pools (**_k_**).
+
+![costfuncpools](https://drive.google.com/uc?export=view&id=1lW93crB3YIFeOd4Y8iG0fxkmKHZKlrgr)
+
+We found that under this CIP, the number of pools needed to attack the ecosystem (**_k<sub>adversarial</sub>_**) increases linearly with the total number of pools (left figure). This observation is in line with the expectation that an ecosystem's security should become more robust as the number of pools increases. Unfortunately, this expectation is **not true** for the protocol currently implemented in Cardano. In addition, notice the existence of a minimum in the plot (right figure) at around 1600 pools. The total pledge to conduct a Sybil attack at this minimum is 45 M ADA in pledge (when **_n_** = 15) and at least 600 pools turning adversarial. The total pledge to conduct a Sybil attack increases as the number of pools either decreases or increases from this minimum. For epoch 321 at **_n_** = 15, the total pledge to conduct a Sybil attack is 52 M ADA in pledge and at least 1108 adversarial pools. We dare say that the total pledge to conduct a Sybil attack under the current protocol is 0 ADA in pledge and only 177 adversarial pools. This number of adversarial pools needed to attack the ecosystem under the current protocol does not change even if the total number of pools increases, unless the saturation limit (68 M) is decreased. These pools can easily get saturated by implementing lucrative (and maybe fake) ISPOs. This mechanism of stake centralization had already been demonstrated during the Sundaeswap ISPO which involved 98 pools that were at or near saturation.
+
+## **_New Reward Structure_**

```

Comment:
While related to the current proposal, it seems that this section is a proposal-in-a-proposal. I would strongly suggest to split it from the current CIP in order to separate the discussion and decisions in two streams. Ideally, CIPs should be focused on _one idea_ so that we can keep the discussion relatively efficient.

Reply:
Hey @KtorZ and @rphair,

So, I've been thinking about revising the proposal, but I do not think that I can separate **a) delegation re-structuring** to **b) reward restructuring**. The modified proposal would be:

1) **delegation structuring**: keep optimal saturation limit as _circulating supply equally divided by the number of pools (φ/k)_" so that small pools can be guaranteed some delegation in order to survive.
2) **reward structuring**: decouple the delegator reward from the operator reward.

- the **delegator reward** is fixed at 4.5% subject to pool performance. If pool performance is 100%, the delegators will get their 4.5%. In a way, delegation is now a vote of confidence rather than a wealth-seeking enterprise. In the first place, the wealth-seeking behavior is what makes the ecosystem centralized.
- the **operator reward** will have a separate equation that's dependent on saturation level, pledge, and performance but that still needs to get formulated. The idea is to make splitting undesirable. For example, a 50M pledge in just one pool will earn 6% reward (so 50M * 0.06), but splitting it into two pools of 25M pledge each will only make it earn 5% (so 2 * 25M * 0.05). Something along that line...

So, in a way the reward re-structuring is based on how you would want the participants to behave: a) you want the pool operators to avoid pool splitting and b) you want the stakers to minimize wealth-seeking behavior. Because they are decoupled, it is now easy to influence their behavior separately.

I am sure this will take a new set of equations very different from CIP-45. My questions are:
1) Will I be able to combine both **delegation restructuring** and **reward restructuring**? Splitting them into different CIPs will not make sense.
2) Do I need to make another post in the Cardano Forum and then re-post a new CIP?

Reply:
> Will I be able to combine both delegation restructuring and reward restructuring? Splitting them into different CIPs will not make sense.

@jycappucino the way you've described it, those two ideas are distinct and could each be implemented separately: so the logical thing would be to have two separate CIPs (by same reasoning as @KtorZ was offering) even if they appear to have the same goal.

> Do I need to make another post in the Cardano Forum and then re-post a new CIP?

The [original Forum posting](https://forum.cardano.org/t/cip-stake-decentralization-using-pledge-as-a-bidding-parameter-to-determine-saturation-limit/95936) linked in the CIP header will form the basis for whatever follow-up proposal you make, and should be continued for that reason alone (note many long standing threads on the Cardano forum have gone on for 2 years or more).

But more importantly the contributors to that thread have included several people interested in this issue (including the author of CIP-50) and the evolution of your idea would be helped by retaining these people rather than fragmenting your audience.

For you to check: whether Discourse will allow you to edit your title, after so much time has gone by since your OP, if terms like "Pledge as a bidding parameter" no longer apply.  If necessary you can get a forum moderator to retitle it for you.